paella-opencast/plugins/es.upv.paella.opencast.loader/01_prerequisites.js

```diff
@@ -155,7 +155,18 @@ paella.opencast = new (Class ({
             if (authResult && authResult.dceReturnStatus) {
                 var returnStatus = authResult.dceReturnStatus;
                 if (("401" == returnStatus || "403" == returnStatus) && authResult.dceLocation) {
+
+                  var redirectDelay = window.parent !== window
+                    ? $("iframe", window.parent.document).index(self.frameElement) * 100
```

Comment:
@lbjay Can you verify if the window.parent.document translates the same way when embedded in a non-browser'd Canvas app (i.e. the iphone app with the WebView).